src/view/actions/sendEvent.jsx

```diff
@@ -181,13 +243,18 @@ const SendEvent = () => {
             <div className="u-gapTop">
               <InfoTipLayout tip="Influences whether the SDK should retrieve and render personalization content, among other things.">
```

Comment:
Maybe remove the `among other things`? Let's run this by Justin & Davis.

Reply:
sure! thank you